core/capabilities/remote/executable/request/client_request.go

```diff
@@ -165,7 +166,7 @@ func newClientRequest(ctx context.Context, lggr logger.Logger, requestID string,
 		cancelFn:                   cancelFn,
 		createdAt:                  time.Now(),
 		requestTimeout:             requestTimeout,
-		requiredIdenticalResponses: int(remoteCapabilityDonInfo.F + 1),
+		requiredIdenticalResponses: aggregation.ByzantineQuorum(len(remoteCapabilityDonInfo.Members), int(remoteCapabilityDonInfo.F)),
```

Comment:
This will not work very well with current Writer DONs, which are non-OCR and have N=9, F=5. We need to either detect non-OCR caps here or adjust the F for those caps.